user_guide/data_management_grids.rst

```diff
@@ -0,0 +1,306 @@
+.. _user-guide-ui-components-grids:
+    
+Grids
+=====
+
+Initially, a grid is an aggregated view of all the records of the same entity. Each row of a grid is one records, each 
+column is one of the grid properties, and thus each cell displays a value of a property for a record. .
+
+For example, you can open a grid of all the Contacts. 
+
+|
+
+.. image:: ./img/data_management/grid/grid_01.png
+
+|
+
+In the example, you can immediately see some major details of our contacts, such as their first and last names, email, 
+phone number, etc.
+
+You can adjust and save grid views, updated and delete grid records, get to the Edit and View pages of any grid record, 
+and even export all the data in the grid as a .csv file.
+
+Get to a Grid
+-------------
+
+There are several ways to get to a grid.
+
+- Select the relevant point in the :ref:`menu bar <user-guide-navigation-menu>`: to get to the *"All Contacts"* grid, 
+  select the *"Contacts"* menu item, to get to the *"All Reports"* grid, click the *"Manage Custom Reports"* menu item, 
+  etc. 
```

Comment:
Edit: Select the relevant point in the :ref:`menu bar <user-guide-navigation-menu>`: For instance, select the *"Contacts"* menu item to get to the *"All Contacts"* grid, click the *"Manage Custom Reports"* menu item to get to the *"All Reports"* grid, and so on.